_data/simple-icons.json

```
@@ -14835,6 +14835,15 @@
             "hex": "010101",
             "source": "https://socket.io"
         },
+        {
+            "title": "Softcatalà",
+            "hex": "ba2626",
+            "source": "https://www.softcatala.org/uploads/2016/03/SC_marca.svg",
+            "guidelines": "https://www.softcatala.org/premsa/imatge-corporativa/",
+            "license": {
+                "type": "CC BY-SA 4.0"
```

The license ID format is wrong. Please use the format below:

```suggestion
                "type": "CC-BY-SA-4.0"
```

You can find all ID from this sheet: https://spdx.org/licenses